### ISSUE TITLE
[Accessibility] WCAG 1.3.1, 4.1.2 -- Editor pane missing label

### ISSUE THREAD

**Issue by xtinae** [swagger-api/swagger-editor#2409]
### Q&A 
 - OS: Windows
 - Browser: Edge, Chrome
 - Version: N/A not browser specific
 - Method of installation: npm
 - Swagger-Editor version: 3.15.2
 - Swagger/OpenAPI version: N/A

### Describe the bug you're encountering
There is no label provided for the editor pane on the swagger editor.
Snippet: `<textarea class="ace_text-input" wrap="off" autocorrect="off" autocapitalize="off" spellcheck="false" style="opacity: 0; height: 16.5px; width: 7.69734px; left: 61px; top: 0px;"></textarea>`

### To reproduce...
Run Accessibility Insights for Web extension (https://accessibilityinsights.io/docs/en/web/overview/) 
As this is a html issue can reproduce here https://editor.swagger.io/

### Expected behavior
There is should be a label provided for the editor pane on the swagger editor.
For proper accessibility, all form elements should have a corresponding label. 

### Additional context or thoughts
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

### How to fix
    Fix any of the following:
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Form element does not have an implicit (wrapped) <label>
      Form element does not have an explicit <label>
      Element has no title attribute
      Element has no placeholder attribute
      Element's default semantics were not overridden with role="none" or role="presentation"

**Comment by MichakrawSB** [comment]
We’ve released Swagger Editor v5! 🎉 Check out the details here: [Inside the New Swagger Editor](https://swagger.io/blog/inside-the-new-swagger-editor/).
We’re closing old issues related to previous versions. If you think any of them are still relevant, please open a new issue – this helps us prioritize what matters most. Thanks for your activity! 🚀
